### DISCUSSION TITLE
Cant create SSL certificate for only one specific subdomain

### DISCUSSION THREAD

**Discussion by ThoreBlesse** [NginxProxyManager/nginx-proxy-manager#3454] [Q&A]
Hello,

I tried setting up a new Proxy Host for the subdomain "sub1.mydomain.net," and it works fine without SSL, accessible from the internet. However, when attempting to add SSL, I consistently receive an "Internal Error" message.

Interestingly, when I performed the same process with another subdomain, "sub2.mydomain.net," SSL certificate creation was successful without any issues.

Both subdomains have CNames pointing to the same main domain.
I'm using a Dyn DNS.

Why am I able to generate an SSL certificate for one domain but encountering failure for the other?

Thank you for any assistance.

**Comment by protonaut** [comment]
I habe the exact same problem.
I used duckdns before and it worked just fine. Now I switched to my own domain, creates subdomains with CNAME pointing. The redirecting through myfritz works just fine.

But the creation of TSL/ certificate courses internal errors in the proxy manager.
Any hints?

**Reply by ThoreBlesse** [reply]
I somehow got it to work and it works now. I can’t explain what exactly I did, but I‘ve disabled IPv6 in my FritzBox DynDNS URL so that the main domain only points to my public v4 address. 
I tried again but still failed.
Frustrated I quit for the day and let the server running for around 18 hours.
I tried it again the next day and it worked for some reason….
